pkg/trie/triedb/print.go

```diff
@@ -0,0 +1,18 @@
+// Copyright 2024 ChainSafe Systems (ON)
+// SPDX-License-Identifier: LGPL-3.0-only
+
+package triedb
+
+import (
+	"fmt"
+
+	"github.com/ChainSafe/gossamer/lib/common"
+)
+
+func (t *TrieDB) String() string {
```

Comment:
Why have this be its own file?

Reply:
Actually it doesn't need its own file, but I was trying to replicate the `inmemory` trie files structure https://github.com/ChainSafe/gossamer/blob/development/pkg/trie/inmemory/print.go

Where we are separating the trie methods into different files